xlml/apis/task.py

```diff
@@ -503,3 +504,118 @@ def clean_up(self, resource: airflow.XComArg, project_id: str, zone: str) -> DAG
       AirflowTaskTimeout: An error occurs when execution_timeout is breached.
     """
     return gpu.delete_resource.override(group_id="clean_up")(resource, project_id, zone)
+
+
+@dataclasses.dataclass
+class GpuGkeTask(BaseTask):
+  """This is a class to set up tasks for GPU.
+
+  Attributes:
+    image_project: the project that an image belongs to.
+    image_family: the family group that an image belongs to.
+  """
+
+  task_test_config: test_config.JSonnetGpuTest
+  task_gcp_config: gcp_config.GCPConfig
+  cluster_name: str
+  job_create_timeout: datetime.timedelta = datetime.timedelta(minutes=10)
+  # TODO: metrics
```

Comment:
Will you be the task owner of this? Basically add post_process right?

Reply:
This resnet test itself doesn't actually write metrics correctly (see the  `--nometrics_debug` flag in the test configs), so I don't actually have a good test case to make sure this works. Otherwise adding this should be straightforward, so whoever adds a new multihost GPU test that writes metrics can enable this.

Reply:
Oh...I did not mean the benchmarking metrics. I was saying the default metric entry into job_history. Basically integrate with post_process step so that in the dashboard (at least for system monitoring purpose), i.e. we know this tests succeeded or failed, and how many tests are running in our repo, etc.

Reply:
Got it. I'll put myself as a task owner for job history metrics.